A14-channelz.md

```diff
@@ -698,32 +755,30 @@ message ServerRef {
   // An optional name associated with the server.
   string name = 6;
   // Intentionally don't use field numbers from other refs.
-  reserved 1, 2, 3, 4;
+  reserved 1, 2, 3, 4, 7, 8;
 }
 
 message Server {
   ServerRef ref = 1;
   ServerData data = 2;
-  ServerChannelTrace trace = 3;
 
-  // The list of all connected sockets refs, possibly including those in
-  // listen_socket.  There are no ordering guarantees.
-  repeated SocketRef socket = 4;
```

Comment:
Why did this go away?

Reply:
@a11r raised concerns about socket pagination.   It was moved into ListServerSockets